neurow/lib/neurow/public_api/endpoint.ex

```diff
@@ -185,11 +185,6 @@ defmodule Neurow.PublicApi.Endpoint do
     {_, message} = first
 
     if message.timestamp > last_event_id do
-      if sent == 0 do
```

Comment:
Seems not required anymore after the refactoring of unit test and integration tests.